plutus-tx/src/PlutusTx/Builtins/Internal.hs

```diff
@@ -10,6 +10,9 @@
 -- they're NOINLINE!
 {-# OPTIONS_GHC -O0 #-}
 
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
```

Comment:
What is this for?

Reply:
Without this option the pragma below is considered "unrecognized" and HLINT emits a warning:

<img width="568" alt="image" src="https://github.com/IntersectMBO/plutus/assets/1009751/b3a82c6e-b218-425e-9958-07b50ceb3275">